projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +169,112 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
+  row.get('td').eq(ID_COLUMN).invoke('text').then((x) => {
+    x = x.substring(ID_DELIMITER);
+    testTicketDetails.id = x.toString();
+  });
+  row.get('td').eq(SUBJECT_COLUMN).invoke('text').then((x) => {
+    x = x.substring(SUBJECT_DELIMITER);
+    testTicketDetails.subject = x.toString();
+  });
+  row.get('td').eq(STATUS_COLUMN).invoke('text').then((x) => {
+    x = x.substring(STATUS_DELIMITER);
+    testTicketDetails.status = x.toString();
+  });
+
+  return testTicketDetails;
+}
+
+export function verifyTicketDetailsByComparingTicketHeaderToExtractedDetails(ticketDetails: TestTicketDetails){
+  //assert title when available
+
+  cy.get('.cx-card-label').eq(0).then((x) => cy.wrap(x).should('include.text', ticketDetails.id));
+  cy.get('.cx-card-label').eq(3).then((x) => cy.wrap(x).should('include.text', ticketDetails.status));
+}
+
+export function createNewTicket(){
+  const testTicketDetails: TestTicketDetails = {
+    subject: 'Entering a subject',
+    message: 'Typing a message',
+    category: TestCategory.complaint,
+  };
+
+  visitElectronicTicketListingPage();
+  openCreateTicketPopup();
+  fillTicketDetails(testTicketDetails);
+  clickSubmit();
+  verifyCreatedTicketDetails(testTicketDetails);
+}
+
+export function verifyTicketStatusIsOpenInTicketDetailsPage(){
+  cy.get('.cx-card-label').eq(3).contains("Open");
+}
+
+export function verifyTicketStatusIsClosedInTicketDetailsPage(){
+  cy.get('.cx-card-label').eq(3).contains("Closed");
+}
+
+export function closeTicket(){
+  cy.get('.btn.btn-block.btn-action').click();
+  cy.get('.cx-customer-ticket-form-body').should('exist');
+  cy.get('label > .form-control').last().type("Closing Ticket");
+  cy.get('.cx-customer-ticket-form-footer > .btn-primary').click();
+}
+
+export function verifyTicketStatusIsOpenOnListingPage(){
+  const row = cy.get('cx-customer-ticketing-list').find('tbody').get('tr').eq(1);
+  row.get('td').eq(5).contains(" Open ");
+}
+
+export function verifyTicketStatusIsClosedOnListingPage(){
+  const row = cy.get('cx-customer-ticketing-list').find('tbody').get('tr').eq(1);
+  row.get('td').eq(5).contains(" Closed ");
+}
+
+export function verifyMessageBoxExists(doesExist){
+  if(doesExist){
+    cy.get('.form-control').should('exist');
+  } else {
+    cy.get('.form-control').should('not.exist');
+  }
+}
+
+export function reopenTicket(){
+  cy.get('.btn.btn-block.btn-action"').click();
+  cy.get('.cx-customer-ticket-form-body').should('exist');
+  cy.get('label > .form-control').last().type("Reopening Ticket");
+  cy.get('.cx-customer-ticket-form-footer > .btn-primary').click();
+}
+
+export function verifyAllCustomerMessagesAreBeingPopulatedInChatHistory(){
+  verifyLastMessageWasPosted();
+}
+
+export function clickSend(){
+  cy.get('.cx-send').click();
+}
+
+export function postMessageAsCustomerIntoChatBox(){
+  cy.get('.form-control').type("Update ticket with comments");
+}
```

Comment:
Please reuse the following function, to replace the above two

https://github.com/SAP/spartacus/blob/679d920a73a8cb7fef2a769e065e2f39713249f0/projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing-helpers/ticket-details.ts#L28